pkg/config/multinode.go

```diff
@@ -0,0 +1,22 @@
+package config
+
+type MultiNode struct {
+	Enabled bool `json:"enabled"`
+	// only one controlplane node is supported
+	// IP address of control plane node
+	Controlplane string `json:"controlplane"`
+}
+
+// MultinodeConfig populates multinode configurations to Config.MultiNode
+func (c *Config) MultinodeConfig(enabled bool) {
+	if !enabled {
+		return
+	}
+	c.MultiNode.Enabled = enabled
+	c.MultiNode.Controlplane = c.Node.NodeIP
+
+	// Use controlplane node IP as APIServer backend (instead of 10.44.0.0)
+	c.ApiServer.AdvertiseAddress = c.Node.NodeIP
```

Comment:
@pacevedom  This overrides the advertiseAddress IP of APIServer for multi-node since the service cdir cannot be accessed by extra worker nodes. Do you see any issue with API certificate by doing so?

Reply:
Yes, we do need some changes for this to work. Let me create the PR.

Reply:
I checked and everything is already there, so no additional changes required.